core/rawdb/accessors_indexes.go

```diff
@@ -63,9 +63,30 @@ func WriteTxLookupEntries(db ethdb.KeyValueWriter, block *types.Block) {
 	}
 }
 
+// WriteTxLookupEntriesByHash is identical to WriteTxLookupEntries, but does not
+// require a full types.Block as input
+func WriteTxLookupEntriesByHash(db ethdb.KeyValueWriter, number []byte, hashes []common.Hash) {
```

Comment:
Period after the comment.

Why is `number` a byte slice. That seems super unorthodox in an API.

Reply:
I have no idea. Fixing